### PR TITLE
Added documentation for unwovenClassesDir param

### PR DESCRIPTION
**Pull request by nesterov-n**
this refers #47 
Example for new parameter unwovenClassDir for site

### CODE REVIEW

**Comment by dmarkov** [comment]
@nesterov-n Many thanks, I will find someone to review it before we merge

**Comment by dmarkov** [comment]
@original-brownbear please review

**Comment by original-brownbear** [comment]
@nesterov-n I'm on it :)

File: src/site/apt/example-unwoven.apt.vm [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ Copy unwoven classes
+ ------
+ Nikolay Nesterov
+ ------
+ 2016-04-15
+ ------
+
+ ~~
+ ~~ #set($sign="$")
+ ~~ Copyright (c) 2012-2015, jcabi.com
```
Comment (original-brownbear) [inline src/site/apt/example-unwoven.apt.vm]: @nesterov-n  `-2016` ;)

File: src/site/apt/example-unwoven.apt.vm [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ ~~ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ ~~ HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ ~~ STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ ~~ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ ~~ OF THE POSSIBILITY OF SUCH DAMAGE.
+ ~~
+
+Copy unwoven classes
+
+  You can make copy of your classes before weaving.
```
Comment (original-brownbear) [inline src/site/apt/example-unwoven.apt.vm]: @nesterov-n You can make **a** copy of your classes ...

File: src/site/apt/example-unwoven.apt.vm [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  Add <<<unwovenClassesDir>>> parameter to plugin configuration.
```
Comment (original-brownbear) [inline src/site/apt/example-unwoven.apt.vm]: @nesterov-n Add **the** <<<unwoven ... to **the** plugin configuration.

**Comment by original-brownbear** [comment]
@nesterov-n just 3 typos/grammar mistakes as far as I can see :)

**Comment by nesterov-n** [comment]
@original-brownbear Thanks! Fixed

**Comment by original-brownbear** [comment]
@nesterov-n thanks!

**Comment by original-brownbear** [comment]
@rultor merge please :)

**Comment by nesterov-n** [comment]
@yegor256 We need your confirmation here

**Comment by original-brownbear** [comment]
@yegor256 ping :)

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by original-brownbear** [comment]
@rultor deploy please.

**Comment by dmarkov** [comment]
@original-brownbear thanks, paid, **24 mins** to your account, payment ID is `AP-5JX0946422753613E`, task took 141 hours and 47 mins to complete... you're getting extra minutes here (c=9)... +24 to your rating, your total score is [+4173](http://www.netbout.com/b/36646?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy pls

**Pull request merged**